docs/sources/troubleshoot/support_bundle.md

```diff
@@ -0,0 +1,46 @@
+---
+canonical: https://grafana.com/docs/alloy/latest/troubleshoot/support_bundle/
+description: Learn how to generate a support bundle
+title: Generate a Support Bundle
+menuTitle: Generate Support Bundle
+weight: 300
+---
+
+# Generate Support Bundle
+
+The `/-/support?duration=N` endpoint returns a 'support bundle', a zip file that contains information
+about a running {{< param "PRODUCT_NAME" >}} instance, and can be used as a baseline of information when trying
+to debug an issue.
+
+{{< admonition type="note" >}}
+This endpoint is enabled by default, but may be disabled using the `--disable-support-bundle` runtime flag.
+{{< /admonition >}}
+
+The duration parameter is optional, must be less than or equal to the
+configured HTTP server write timeout, and if not provided, defaults to it.
+The endpoint is only exposed to the {{< param "PRODUCT_NAME" >}} HTTP server listen address, which
+defaults to `localhost:12345`.
+
+The support bundle contains all information in plain text, so you can
+inspect it before sharing to verify that no sensitive information has leaked.
+
+In addition, you can inspect the [supportbundle implementation](https://github.com/grafana/alloy/tree/internal/service/http/supportbundle.go)
+to verify the code used to generate these bundles.
+
+A support bundle contains the following data:
+* `alloy-components.json` contains information about the [components][components] running on this {{< param "PRODUCT_NAME" >}} instance, generated by the 
+`/api/v0/web/components` endpoint.
+* `alloy-logs.txt` contains the logs during the bundle generation.
+* `alloy-metadata.yaml` contains the {{< param "PRODUCT_NAME" >}} build version and the installation's operating system, architecture, and uptime.
+* `alloy-metrics.txt` contains a snapshot of {{< param "PRODUCT_NAME" >}}'s internal metrics.
+* `alloy-peers.json` contains information about the identified cluster peers of this {{< param "PRODUCT_NAME" >}} instance, generated by the 
+`/api/v0/web/peers` endpoint.
+* `alloy-runtime-config.txt` contains the values of the runtime flags available in {{< param "PRODUCT_NAME" >}}.
```

Comment:
config is a bit misleading because it could be confused with the alloy config

Reply:
Sure, `runtime-flags` would probably be clearer.